src/App.js

```diff
@@ -9,7 +10,19 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>
+      <Article title={article} />
+    </h1>
+
+    <span>
+      <Article date={article} />
+    </span>
+
+    <p>
+      <Article text={article} />
+    </p>
```

Comment:
it should be one Article
Don't pass all object `article` to component, pass only values from the object